### ISSUE TITLE
After wrapped app is crashed

### ISSUE THREAD

**Issue by bhumin3i** [msintuneappsdk/intune-app-wrapping-tool-ios#25]
**Describe the bug:**

**To Reproduce**
1. Go to 'ios device'
2. Click on 'app icon'
3. See error

![Image from iOS (7)](https://user-images.githubusercontent.com/25737987/59247876-6a40bf80-8c3e-11e9-9e8d-fcdcacc9026d.png)

**Expected behavior:**
open authenticator when download from company portal, enter credential and get access token

**Screenshots and logs:**
- If applicable, add screenshots to help explain your problem. 
- If your app is crashing post-wrapping, do you have app logs for the crash itself? 
- If your app is crashing pre-wrapping, do you have app logs for the wrapper errors? 

**Smartphone (please complete the following information):**
 - Device: [iPhone7, 6plus]
 - OS: [e.g. iOS12 and above]

**Intune app wrapping tool (please complete the following information):**
- V11.0.3
 Are you using the latest version?
  Yes
- What platform is your app based in?
 Cordova
- does the app build without being wrapped?
Yes
- For post-wrapping errors, does the app launch without being wrapped?
Yes
- lib https://github.com/AzureAD/azure-activedirectory-library-for-objc
- pod 'ADAL', '2.6.5'

**Comment by Kyle-Reis** [comment]
Hey @bhumin3i, it's not clear to me what the issue is here. The alert in the screenshot you shared is expected for wrapped applications. After the user taps "OK", they should be prompted to enter their credentials. Are you seeing something different, or expecting different behavior?

**Comment by bhandaribhumin** [comment]
@Kyle-Reis yes after use correct certificate and correct sha-1 key now it’s working but every time it’s prompt this popup after click okay they prompt enter credentials.
Also our admin face new issue login is going infinitely.

**Comment by bhumin3i** [comment]
@Kyle-Reis i don't know but today i got new error enroll now!!!! yesterday app was working but today it's not........

**Comment by Kyle-Reis** [comment]
Could you describe what the error is?

**Comment by bhandaribhumin** [comment]
@Kyle-Reis without wrapping I upload into Intune it’s working but with wrapping it’s not.
Message seems like you need to enrolled your device.
Or used different credentials button show enrolled now

**Comment by Kyle-Reis** [comment]
Could you share the exact error message, and maybe a screenshot?

**Comment by bhandaribhumin** [comment]
Actually I try to upload wrapping ipa into company Portal and try again!!! Thanks for instant reply You r the best bro🤘🏻

**Comment by Kyle-Reis** [comment]
Hey @bhandaribhumin, just want to verify you're unblocked before closing this issue out.

**Comment by Kyle-Reis** [comment]
Closing out this issue.
